src/imitation/scripts/common/demonstrations.py

```diff
@@ -91,9 +99,8 @@ def load_expert_trajs(
     Raises:
         ValueError: There are fewer trajectories than `n_expert_demos`.
     """
-    # wrap with str() in case rollout_path is a Path object
-    if "huggingface" in str(rollout_path):
-        rollout_path = expert.download_expert_rollouts(rollout_path=str(rollout_path))
+    if "huggingface" in rollout_type:
```

Comment:
```suggestion
    if rollout_type.endswith("-huggingface"):
```

Reply:
Also consider checking that previous `rollout_path=None`, so users don't set rollout_path to something non-None and expect it to have an effect and be surprirsed?

Reply:
Have raised a warning for such a case. Let me know if this feels appropriate or if we should do something else like raising an error.